.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on: [pull_request]
+
+env:
+    CARGO_TERM_COLOR: always
+
+jobs:
+  shellcheck:
+    name: Check / shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          fail_on_error: true
+          path: "."
+          pattern: "*.sh"
```

Comment:
This is the default value. Please remove this line.

Reply:
Removed the default value